docs/api/menu-item.md

```diff
@@ -145,10 +185,15 @@ will turn off that property for all adjacent items in the same menu.
 
 You can add a `click` function for additional behavior.
 
-#### `menuItem.label`
```

Comment:
Why are we removing the `label` docs?

I agree that we should remove the docs for `click`.  The `click` property isn't actually the `click` function you pass to the constructor but `label` appears to both exist and be correct 👍